### ISSUE TITLE
scipy.integrate has no attribute trapz

### ISSUE THREAD

**Issue by farzanehnakhaee70** [LCAV/pyroomacoustics#362]
In scipy version of 1.14.1, there is no trapz in integrate. Therefore, after upgrading scipy, I got an error from [this](https://github.com/LCAV/pyroomacoustics/blob/master/pyroomacoustics/denoise/iterative_wiener.py#L340) part of the code:

```
AttributeError: module 'scipy.integrate' has no attribute 'trapz'
```

**Comment by fakufaku** [comment]
@farzanehnakhaee70 Thanks for reporting this bug.

It seems the function `trapz` has been deprecated in favor of `trapezoid` in a recent version of scipy.
I will fix the package accordingly.

**Comment by fakufaku** [comment]
Fixed in #363 and [v0.7.7](https://github.com/LCAV/pyroomacoustics/releases/tag/v0.7.7)
